hardhat.config.ts

```diff
@@ -35,13 +35,26 @@ task('accounts', 'Prints the list of accounts', async (args, hre) => {
 
 const config: HardhatUserConfig = {
   solidity: {
-    version: '0.8.6',
-    settings: {
-      optimizer: {
-        enabled: true,
-        runs: 10,
+    compilers: [
+      {
+        version: '0.8.6',
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 10,
+          },
+        },
       },
-    },
+      {
+        version: '0.8.9',
```

Comment:
two different compilers ? :)

Reply:
I wrote the zodiac module in 8.6 as that is what the rest of the gnosis zodiac stuff was written with. Not really necessary though, 1 compiler would be cleaner